src/pages/en/core-concepts/astro-components.md

```diff
@@ -273,12 +273,19 @@ In Astro, you use the standard `kebab-case` format for all HTML attributes inste
 
 #### Comments
 
-In Astro, you can use standard HTML comments where JSX would use JavaScript style comments.
+In Astro, you can use standard HTML comments or JavaScript-style comments.
 
-```html title="example.astro"
+```astro title="example.astro"
+---
+---
 <!-- HTML comment syntax is valid in .astro files -->
+{/* JS comment syntax is also valid */}
 ```
 
+:::caution
+HTML-style comments will be included in browser DOM, while JS ones will be skipped. To leave TODO messages or other development-only explanations, you may wish to use JavaScript-style comments instead.
```

Comment:
```suggestion
HTML-style comments will be included in the browser DOM when compiled, while JS ones will be skipped. To leave TODO messages or other development-only explanations, you may wish to use JavaScript-style comments instead.
```
Would you say this phrase is correct, to describe non-raw HTML only? I'm trying to think of the super-shortest thing we can say that's both true and helpful!

Reply:
Yes, your version is better one! We may commit it.

Reply:
Would you mind merging that in too?

ah, I see we have merged this into main 🙂